### DISCUSSION TITLE
[FEATURE REQUEST]Emulation of some old LapTop

### DISCUSSION THREAD

**Discussion by jvernet** [86Box/86Box#1723] [Ideas]
If possible emulation of old MSDOS (mostly 8088/Nec V20/8086) LapTop like:
Tandy 1100 & 1400
Sharp PC-3000
HP 95/100/200 Lx
GRiD
Olivetti M111, M15,M22,
Bondwell B2200
IBM 5155, and 5140 "Convertible"

But Also Compaq Portable 386, 486. Zenith... HP...

Some have great display to emulate/render (the yellow or red screen o the GRiD). Or function (MSDOS in ROM).

For the next version !

**Comment by chrominancewastaken** [comment]
I was thinking of emulating other Toshiba laptops like the T5200C and the Satellite 200CDS/CDT.
